app/controllers/admins_controller.rb

```diff
@@ -6,12 +7,14 @@ def show
 
   def new
     @admin = Admin.new
+    authorize! :new, @admin
   end
 
   def create
     @admin = Admin.new(admin_params)
+    authorize! :create, @admin
     if @admin.save
-      session[:admin_id] = @admin.id
+      # session[:admin_id] = @admin.id  // Uncomment if we want to login the new admin apon creation.
```

Comment:
Im leaving this line to long so that its knotist and a convo can be had.  I say merg it and catch it next time around...